ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -291,6 +292,12 @@ StatusSectionLayout {
             amISectionAdmin: root.amISectionAdmin
             sendViaPersonalChatEnabled: root.sendViaPersonalChatEnabled
             paymentRequestFeatureEnabled: root.paymentRequestFeatureEnabled
+            selectingMembers: headerContentLoader.item && !!headerContentLoader.item.selectingMembers
+            onGroupUpdated: {
+                if (headerContentLoader.item && headerContentLoader.item instanceof ChatHeaderContentView) {
```

Comment:
This seems somewhat odd... to rely on the concrete underlying type of the component; but I don't really have a better idea atm :shrug:

Reply:
Yeah, it was either that or doing `!!headerContentLoader.item.groupUpdated`. It's as you prefer

Reply:
Is this condition needed at all? It's already guarded by `!!headerContentLoader.item.selectingMembers` which will be evaluated to false when it's not instance of `ChatHeaderContentView`, preventing emiting of that signal.